src/Apps/Artist/Routes/Consign/Utils/getConsignmentData.ts

```diff
@@ -0,0 +1,2593 @@
+/**
+ * Passed https://docs.google.com/spreadsheets/d/1EicHOCyCozsPBkVOkQGTTUtEds7qQuacUAgLy-C2-Nk/edit#gid=0
+ * through a CSVtoJSON converter
+ */
+
+import { flatten, groupBy } from "lodash"
+
+export interface ArtistConsignment {
+  artworks: object[]
+  metadata: {
+    highestRealized: string
+    realized: string
+    recentlySoldArtworkIDs: string[]
+    roundedUniqueVisitors: string
+    roundedViews: string
+    str: string
+    uniqueVisitors: string
+    views: string
+  }
+}
+
+/**
+ * Massage data to be more usable
+ */
+export function getConsignmentData(pathname: string): ArtistConsignment {
+  const dataGroupedByURL = groupBy(getCSVData(), "url")
+
+  const mappedData = Object.entries(dataGroupedByURL).reduce(
+    (acc, [key, value]) => {
+      /**
+       * The first item in the array contains metrics related to artist; the rest
+       * are artworks to display
+       */
+      const [meta, ...artworks] = value
+
+      const recentlySoldArtworkIDs = flatten(
+        artworks.map(artwork => {
+          const ids = artwork[
+            "Artwork ids (recently sold) (comma separated)"
+          ].split(",")
+          return ids
+        })
+      )
+
+      // Mapped CSV columns
+      const {
+        "views to content from that artist (last month)": views,
+        "ROUNDED views to content from that artist (last month)": roundedViews,
+        "unique visitors to content from artist (last month)": uniqueVisitors,
+        "ROUNDED unique visitors to content from artist (last month)": roundedUniqueVisitors,
+        "STR (last 12 months)": str,
+        "Realized / estimate": realized,
+        "Highest realized price": highestRealized,
```

Comment:
@jonallured, @pepopowitz - I'm taking the columns straight out of Joannas spreadsheet via CSV converter, see any problem with this?

Reply:
One thought I have is about future updates. I don't know what our thoughts are on how much this data is going to change, but given past experience with spreadsheet-driven-data, I wouldn't count on what's in the spreadsheet being final. 

Assuming the data is going to change at the source, we should have a simple & repeatable way of replacing it here. I could see that going a few ways:

* Including an actual raw CSV (or something) in the project and parsing that via this function. Replacement of data would be as simple as exporting the spreadsheet as CSV and pasting it into the file.
* Leaving it as JSON, but providing guidance on which tool to use to convert the CSV to JSON. I know this seems simple, but I think it's important to provide this guidance so that it's repeatable, and people don't have to investigate tooling.
* Including a task in the project to turn the CSV into JSON. 

Regardless, I think it'd be important to include docs in a .md about how to replace the data. I also think if we stick with the JSON route, we should extract the JSON to a file completely on its own. I find large, hardcoded data sets easier to work with when they stand alone.

Reply:
All good points! Will address.

(I didn't want to include a CSV-to-JSON lib in the project because it would be a 1-off and unsure how often this data will need to change, but I do think moving data into its own file would be helpful, and adding some instructions. If this starts to change with any regularity then we can pull in a lib.)

Reply:
fwiw, I like how you addressed this @damassi 💯 

My only worry is that they'll change the column names or order or something, haha.